sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/data/MutableLongPointData.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.internal.data;
+
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.sdk.metrics.data.LongExemplarData;
+import io.opentelemetry.sdk.metrics.data.LongPointData;
+import java.util.Collections;
+import java.util.List;
+import javax.annotation.Nullable;
+
+public class MutableLongPointData implements LongPointData {
```

Comment:
Add missing internal boilerplate:
```
 * <p>This class is internal and is hence not for public use. Its APIs are unstable and can change
 * at any time.
```
Also, we'll want to implement `int hashCode()` and `boolean equals(Object)` since these are record classes.

This same advice applies to `MutableDoublePointData`.

Reply:
Added